tests/nextflow.config

```diff
@@ -0,0 +1,47 @@
+// Default config file that nf-test relies on
+// Specify resources for processes based on labels
+process {
+    // Single-core processes
+    withLabel: single {
+        cpus = 1
+        memory = 4.GB
+    }
+
+    withLabel: single_large_memory {
+        cpus = 1
+        memory = 15.GB
+    }
+
+    withLabel: single_xlarge_memory {
+        cpus = 1
+        memory = 15.GB
```

Comment:
Should be bigger?

Reply:
I'll add a comment, but the github free vm's that are provided with github actions have 4 cpus and 16 GB of memory so we must cap everything.